pkg/scheduler/cache/status_updater/default_status_updater.go

```diff
@@ -107,6 +109,9 @@ func (su *defaultStatusUpdater) Evicted(
 		message)
 }
 
+// +kubebuilder:rbac:groups="",resources=pods,verbs=create;update;patch
```

Comment:
also create?

Reply:
It has create.. what do you mean?

Reply:
I meant that I don't think the scheduler creates pods, so why does it need this permission?

Reply:
I am sure you right!
Its leftover from yesterday that this line was about pods + pod/status.

Changed!